### PR TITLE
style: Upload actions should not wrap

### LINKED ISSUE #47891: Upload 操作图标换行了
### Reproduction link

[Edit on StackBlitz](https://stackblitz.com/edit/react-5artf4?file=demo.tsx)

### Steps to reproduce

看 demo

image

### What is expected?

不换行

### What is actually happening?

换行

| Environment | Info |
| --- | --- |
| antd | 5.15.2 |
| React | latest |
| System | mac |
| Browser | chrome |

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

close https://github.com/ant-design/ant-design/issues/47891

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Fix Upload file list actions wrap style.         |
| 🇨🇳 Chinese |  修复 Upload 列表行动点换行的问题。         |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/upload/style/list.ts [hunk 1]
```diff
@@ -38,6 +38,8 @@ const genListStyle: GenerateStyle<UploadToken> = (token) => {
           },
 
           [actionsCls]: {
+            whiteSpace: 'nowrap',
+
```
Comment (li-jia-nan) [inline components/upload/style/list.ts:42]: 多余空行

Comment (afc163) [inline components/upload/style/list.ts:42]:
```suggestion
```

File: components/upload/style/list.ts [hunk 2]
```diff
@@ -38,6 +38,7 @@ const genListStyle: GenerateStyle<UploadToken> = (token) => {
           },
 
           [actionsCls]: {
+            whiteSpace: 'nowrap',
```
Comment (afc163) [inline components/upload/style/list.ts:41]:
```suggestion
            whiteSpace: 'nowrap',

```

File: components/upload/style/list.ts (hunk 1 above)
Comment (afc163) [reply components/upload/style/list.ts:42]: 不多余，和下面隔开。

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**